spec/jobs/record_claim_event_job_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+require "geckoboard"
+
+RSpec.describe RecordClaimEventJob do
+  let(:claims) { create_list(:claim, 5, :submitted) }
+
+  subject { described_class.new }
+
+  it "sends each claim's reference, policy and submitted date to claims.submitted.ENV dataset" do
```

Comment:
This makes it sound like `"claims.submitted.ENV"` is a constant in the behaviour, when in fact the `"submitted"` part comes from the second parameter passed to the `perform` method. Can you rewrite this spec to make it clearer that the second parameter is used to determine the dataset?